### PR TITLE
fix(select): only animate placeholder when no selection

### PR DESCRIPTION
**Pull request by kara**
Uses CSS to float the placeholder when the value is set programmatically so no animation plays.  Will only play animation if the panel is opening or closing without a selection.

r: @jelbourn

### CODE REVIEW

File: src/lib/select/select.scss [hunk 1]
```diff
@@ -30,11 +30,22 @@ md-select {
 }
 
 .md-select-placeholder {
+  position: relative;
   padding: 0 2px;
   transform-origin: left top;
 
+  &.md-floating-placeholder {
```
Comment (jelbourn) [inline src/lib/select/select.scss]: Need comment explaining why this is duplicated from the animation code.

**Comment by kara** [comment]
@jelbourn Added the comment!

**Comment by jelbourn** [comment]
LGTM

File: src/lib/select/select.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <div class="md-select-trigger" overlay-origin (click)="toggle()" #origin="overlayOrigin" #trigger>
-  <span class="md-select-placeholder" [@transformPlaceholder]="_getPlaceholderState()"> {{ placeholder }} </span>
+  <span class="md-select-placeholder" [class.md-floating-placeholder]="this._selected"
```
Comment (tinayuangao) [inline src/lib/select/select.html]: `this._selected` is private. Better to use `this.selected`.

Comment (kara) [inline src/lib/select/select.html]: Whoops! You're absolutely right, it should be `selected`.

**Comment by kara** [comment]
@tinayuangao Should be corrected now. Thanks for catching!

**Pull request merged**